src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  function partnerPhrase(gender) {
+    if (gender === 'f') {
+      return `My husband's name is ${partnerName}`;
+    }
+
+    return `My wife's name is ${partnerName}`;
+  }
```

Comment:
there is no need to use function here. use ternary operator and assign the result to a variable and then render a massage about marital status in layout.